Misc/NEWS.d/next/Documentation/2018-04-08-19-09-22.bpo-25735.idVQBD.rst

```diff
@@ -0,0 +1 @@
+Added documentation for func factorial to indicate that returns integer values
```

Comment:
Do we really need a `NEWS` entry for adding 3 words to the doc? I don't think so.